README.md

```diff
@@ -38,7 +38,8 @@ Supported commands (leading slash can be omitted, lower/upper case do not matter
 
 - `/help` - List available commands (you can find details there)
 
-- `/key <KEYS>` (**Xorg only**) - Generate keypress event for a key or a shortcut.
+- `/key [<ARGS>] <KEYS> [<KEYS>...]` (**Xorg only**) - Generate keypress event for a key,
+  a shortcut, or a sequence of them. Additional arguments are forwarded to `xdotool key`.
```

Comment:
IMO, needs more clarification on how the `/key` command works (it should be clear and obvious that all the arguments separated by spaces are forwarded to `xdotool key`)